packages/rancher-windows-gmsa/charts/Chart.yaml

```diff
@@ -0,0 +1,24 @@
+annotations:
+  catalog.cattle.io/auto-install: rancher-windows-gmsa-crd=match
+  catalog.cattle.io/certified: rancher
+  catalog.cattle.io/display-name: Windows GMSA
+  catalog.cattle.io/os: windows
+  catalog.cattle.io/provides-gvr: windows.k8s.io.gmsacredentialspecs/v1
+  catalog.cattle.io/release-name: rancher-windows-gmsa
```

Comment:
Missing annotations:
- `catalog.cattle.io/namespace`
- `catalog.cattle.io/rancher-version`
- `catalog.cattle.io/kube-version`

Reply:
especially since we only support 1.21+

Reply:
@aiyengar2 anytime I include these annotations my chart isn't visible in the UI.

Reply:
@phillipsj that might have to do with your local dev setup or it could be a bug with Rancher (specifically when parsing these annotations in Windows clusters maybe?). Nevertheless, we still need these annotations set on the chart.

@PennyScissors might be able to help you look into why these annotations aren't working since he's familiar with some of this code.